olp-cpp-sdk-core/include/olp/core/utils/Dir.h

```diff
@@ -105,6 +158,17 @@ class CORE_API Dir {
    * @return \c true if any file with the given path exists, \c false otherwise.
    */
   static bool FileExists(const std::string& file_path);
+
+  /**
+   * @brief Change permisions of the directory and all it's content.
+   *
+   * @param path Path to the directory.
+   * @param permisions_mask PermisionsMask mask
+   *
+   * @return \c true if any file with the given path exists, \c false otherwise.
+   */
+  static bool ChangePermisions(const std::string& path,
```

Comment:
If it is only needed for testing then I would prefer to have an anonymous function added in testing and not in public API.

Reply:
Done